internal/pkg/telemetry/meter.go

```diff
@@ -5,26 +5,32 @@ import (
 )
 
 type Meter interface {
-	Counter(name, desc, unit string, opts ...metric.Int64CounterOption) metric.Int64Counter
-	UpDownCounter(name, desc, unit string, opts ...metric.Int64UpDownCounterOption) metric.Int64UpDownCounter
-	Histogram(name, desc, unit string, opts ...metric.Float64HistogramOption) metric.Float64Histogram
+	IntCounter(name, desc, unit string, opts ...metric.Int64CounterOption) metric.Int64Counter
+	IntUpDownCounter(name, desc, unit string, opts ...metric.Int64UpDownCounterOption) metric.Int64UpDownCounter
+	IntHistogram(name, desc, unit string, opts ...metric.Int64HistogramOption) metric.Int64Histogram
+	FloatHistogram(name, desc, unit string, opts ...metric.Float64HistogramOption) metric.Float64Histogram
```

Comment:
Why did we change the naming? I would inderstand that if we have `IntCounter` and `FloatCounter` but we have just 1.
For Histogram it makes sense.

Reply:
In my opinion, it is a good step for the future.

Reply:
Then it would be good to add other methods as well, it's just 1 line of code as preparation.